doc/user/content/guides/top-k.md

```diff
@@ -1,21 +1,13 @@
 ---
-title: "Idioms"
-description: "Recommended SQL idioms."
-weight: 12
+title: "Top K by Group"
+description: "Filter out all but the top elements in a group."
```

Comment:
Could "Find the top- or bottom-K elements in a group." be a simpler and more straightforward definition of Top K? Whenever I've used this function, my goal was always to **find** the top or bottom K elements, so describing it as "filter out all but the top elements" reads a bit weird.